server/api.js

```diff
@@ -12,31 +12,30 @@ router.get("/", (_, res) => {
 
 router.post("/login", (req, res) => {
 	const loginObject = req.body;
-	if (loginObject !== { email: "", password: "" }) {
-		client
-			.query(
-				`
-				SELECT * 
-				FROM users 
-				WHERE user_email=$1 AND user_password=$2
-				`,
-				[loginObject.email, loginObject.password]
-			)
-			.then((response) => {
-				if (response.rowCount < 1) {
-					res.status(400).send("Email or password incorrect");
-				} else {
-					res.send("student");
-				}
-			})
-			.catch((error) => {
-				console.error(error);
-				res.status(error.status).send(error);
+	console.log(loginObject);
+	client
+		.query(
+			`
+			SELECT * 
+			FROM users 
+			WHERE user_email=$1 AND user_password=$2
+			`,
+			[loginObject.email, loginObject.password]
```

Comment:
Looks good, but the password in the db is hashed and needs to be compared accordingly. Please check this: https://github.com/kelektiv/node.bcrypt.js#to-check-a-password